src/views/asset-edit-view/asset-edit-view.tsx

```diff
@@ -6,14 +6,19 @@ import { assetAdminAuthGroups } from "../../services/config/config";
 import { AssetEditLayout } from "./layout";
 
 import { useAsset } from "@mtfh/common/lib/api/asset/v1";
+import { Tenure, useTenure } from "@mtfh/common/lib/api/tenure/v1";
+
 import { isAuthorisedForGroups } from "@mtfh/common/lib/auth";
 import { Center, ErrorSummary, Spinner } from "@mtfh/common/lib/components";
+import { assetHasFloors } from "utils/asset-type";
 
 export const AssetEditView = (): JSX.Element => {
   const { assetId } = useParams<{ assetId: string }>();
 
   const { data: asset, ...assetRequest } = useAsset(assetId);
 
+  const tenure = useTenure(asset?.tenure ? asset?.tenure.id : null).data;
```

Comment:
Can you confirm you've manually tested this? I'm referring to the accessing `data`. I'm thinking this relies on the successful response of an API call, so `data` may not always be promptly available. I wonder if making it `?.data` could prevent that potential issue.